test/integration/firestore.spec.ts

```diff
@@ -76,7 +78,7 @@ describe('admin.firestore', () => {
       .then((snapshot) => {
         const data = snapshot.data();
         expect(data.timestamp).is.not.null;
-        expect(data.timestamp instanceof Date).is.true;
+        expect(data.timestamp instanceof admin.firestore.Timestamp).is.true;
```

Comment:
This is: `expect(data.timestamp).to.be.instanceOf(admin.firestore.Timestamp)`

Reply:
Done. 

I believe the package-lock.json changes are because we define a lot of our dependencies loosely with wildcards. So everytime we regen the file, there are new versions to pick up.